furiosa/models/types.py

```diff
@@ -145,14 +141,23 @@ def tensor_name_to_range(self) -> Dict[str, List[float]]:
         calib_yaml = resolve_source(self._artifact_name, EXT_CALIB_YAML)
         return yaml.full_load(calib_yaml)
 
-    def model_source(self, num_pe: int = 2) -> bytes:
+    def model_source(self, num_pe: [1|2] = 2) -> bytes:
```

Comment:
이거 제가 그냥 예시로 적어놓은 type hint라 valid 하지 않은 타입 힌트인 것 같습니다.
```suggestion
from typing import Literal
...
    def model_source(self, num_pe: Literal[1, 2] = 2) -> bytes:
```